source/includes/connect/mongoclient.c

```diff
@@ -0,0 +1,23 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main (void) {
+
+  // start-connect-to-atlas-w-uri
+  mongoc_client_t *client;
+  mongoc_database_t *database;
+  char *uri;
+
```

Comment:
```suggestion
```
The C Driver no longer supports C89/C90 compatibility, so mixing declarations and code is permitted and preferred.

```c
// Initialize the Mongo C Driver.
mongoc_init();

// Create a new client and connect to the server.
const char *uri = "...";
mongoc_client_t *client = mongoc_client_new (uri);
mongoc_database_t *database = mongoc_client_get_database (client, "admin");
```